### ISSUE TITLE
Compilcation fails on Windows 7 x64

### ISSUE THREAD

**Issue by vkorobkov** [warmuuh/libsass-maven-plugin#13]
Hello, I am using the plugin in the following way(very standard way I would say):

```xml
<plugin>
                <groupId>com.github.warmuuh</groupId>
                <artifactId>libsass-maven-plugin</artifactId>
                <version>${libsass-maven-plugin.version}</version>
                <executions>
                    <execution>
                        <phase>generate-resources</phase>
                        <goals>
                            <goal>compile</goal>
                        </goals>
                    </execution>
                </executions>
                <configuration>
                    <!--<imgPath>../img</imgPath>-->
                    <inputPath>${project.basedir}/src/main/resources/dynamic/scss</inputPath>
                    <outputPath>${project.build.outputDirectory}/static/css</outputPath>
                    <!--<includePath>${basedir}/src/main/sass/plugins/</includePath>-->
                </configuration>
            </plugin>
```

And it works fine under the linux. But on windows it fails and says:

<pre>
[ERROR] Failed to execute goal com.github.warmuuh:libsass-maven-plugin:0.1.4-libsass_3.2.4:compile (default) on project web: Execution default of goal com.github.warmuuh:libsass-maven-plugin:0.1.4-libsass_3.2.4:compile failed: An API incompatibility was encounte
red while executing com.github.warmuuh:libsass-maven-plugin:0.1.4-libsass_3.2.4:compile: java.lang.UnsatisfiedLinkError: Unable to load library 'sass': Native library (win32-x86/sass.dll) not found in resource path ([file:/C:/Users/gcc/.m2/repository/com/github/
warmuuh/libsass-maven-plugin/0.1.4-libsass_3.2.4/libsass-maven-plugin-0.1.4-libsass_3.2.4.jar, file:/C:/Users/gcc/.m2/repository/net/java/dev/jna/jna/4.1.0/jna-4.1.0.jar, file:/C:/Users/gcc/.m2/repository/com/nativelibs4java/jnaerator-runtime/0.12/jnaerator-runt
ime-0.12.jar, file:/C:/Users/gcc/.m2/repository/com/nativelibs4java/ochafik-util/0.12/ochafik-util-0.12.jar, file:/C:/Users/gcc/.m2/repository/javax/enterprise/cdi-api/1.0/cdi-api-1.0.jar, file:/C:/Users/gcc/.m2/repository/javax/annotation/jsr250-api/1.0/jsr250-
api-1.0.jar, file:/C:/Users/gcc/.m2/repository/javax/inject/javax.inject/1/javax.inject-1.jar, file:/C:/Users/gcc/.m2/repository/com/google/guava/guava/10.0.1/guava-10.0.1.jar, file:/C:/Users/gcc/.m2/repository/com/google/code/findbugs/jsr305/1.3.9/jsr305-1.3.9.
jar, file:/C:/Users/gcc/.m2/repository/org/sonatype/sisu/sisu-guice/3.1.0/sisu-guice-3.1.0-no_aop.jar, file:/C:/Users/gcc/.m2/repository/aopalliance/aopalliance/1.0/aopalliance-1.0.jar, file:/C:/Users/gcc/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.inject/0
.0.0.M5/org.eclipse.sisu.inject-0.0.0.M5.jar, file:/C:/Users/gcc/.m2/repository/org/eclipse/aether/aether-util/0.9.0.M2/aether-util-0.9.0.M2.jar, file:/C:/Users/gcc/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.19/plexus-interpolation-1.19.jar, file:
/C:/Users/gcc/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.15/plexus-utils-3.0.15.jar, file:/C:/Users/gcc/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar, file:/C:/Users/gcc/.m2/repository/org/s
onatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar, file:/C:/Users/gcc/.m2/repository/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar])
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>com.github.warmuuh:libsass-maven-plugin:0.1.4-libsass_3.2.4
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/C:/Users/gcc/.m2/repository/com/github/warmuuh/libsass-maven-plugin/0.1.4-libsass_3.2.4/libsass-maven-plugin-0.1.4-libsass_3.2.4.jar
[ERROR] urls[1] = file:/C:/Users/gcc/.m2/repository/net/java/dev/jna/jna/4.1.0/jna-4.1.0.jar
[ERROR] urls[2] = file:/C:/Users/gcc/.m2/repository/com/nativelibs4java/jnaerator-runtime/0.12/jnaerator-runtime-0.12.jar
[ERROR] urls[3] = file:/C:/Users/gcc/.m2/repository/com/nativelibs4java/ochafik-util/0.12/ochafik-util-0.12.jar
[ERROR] urls[4] = file:/C:/Users/gcc/.m2/repository/javax/enterprise/cdi-api/1.0/cdi-api-1.0.jar
[ERROR] urls[5] = file:/C:/Users/gcc/.m2/repository/javax/annotation/jsr250-api/1.0/jsr250-api-1.0.jar
[ERROR] urls[6] = file:/C:/Users/gcc/.m2/repository/javax/inject/javax.inject/1/javax.inject-1.jar
[ERROR] urls[7] = file:/C:/Users/gcc/.m2/repository/com/google/guava/guava/10.0.1/guava-10.0.1.jar
[ERROR] urls[8] = file:/C:/Users/gcc/.m2/repository/com/google/code/findbugs/jsr305/1.3.9/jsr305-1.3.9.jar
[ERROR] urls[9] = file:/C:/Users/gcc/.m2/repository/org/sonatype/sisu/sisu-guice/3.1.0/sisu-guice-3.1.0-no_aop.jar
[ERROR] urls[10] = file:/C:/Users/gcc/.m2/repository/aopalliance/aopalliance/1.0/aopalliance-1.0.jar
[ERROR] urls[11] = file:/C:/Users/gcc/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.inject/0.0.0.M5/org.eclipse.sisu.inject-0.0.0.M5.jar
[ERROR] urls[12] = file:/C:/Users/gcc/.m2/repository/org/eclipse/aether/aether-util/0.9.0.M2/aether-util-0.9.0.M2.jar
[ERROR] urls[13] = file:/C:/Users/gcc/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.19/plexus-interpolation-1.19.jar
[ERROR] urls[14] = file:/C:/Users/gcc/.m2/repository/org/codehaus/plexus/plexus-utils/3.0.15/plexus-utils-3.0.15.jar
[ERROR] urls[15] = file:/C:/Users/gcc/.m2/repository/org/codehaus/plexus/plexus-component-annotations/1.5.5/plexus-component-annotations-1.5.5.jar
[ERROR] urls[16] = file:/C:/Users/gcc/.m2/repository/org/sonatype/plexus/plexus-sec-dispatcher/1.3/plexus-sec-dispatcher-1.3.jar
[ERROR] urls[17] = file:/C:/Users/gcc/.m2/repository/org/sonatype/plexus/plexus-cipher/1.4/plexus-cipher-1.4.jar
[ERROR] Number of foreign imports: 1
[ERROR] import: Entry[import  from realm ClassRealm[maven.api, parent: null]]
</pre>

The software:
1. Windows 7 SP1 x64 Ultimate.

2. java version "1.8.0_20"
Java(TM) SE Runtime Environment (build 1.8.0_20-b26)
Java HotSpot(TM) 64-Bit Server VM (build 25.20-b23, mixed mode)

3. Apache Maven 3.2.5 

Any suggestions?

Many thanks,
Vladimir Korobkov

**Comment by warmuuh** [comment]
are you sure, that it uses.a 64bit jvm? because it tries to load win32-x86/sass.dll, which is not included... i just included 64bit win/linux binaries.

**Comment by vkorobkov** [comment]
Yep you were right. My maven was running on x86 JVM.

But anyway, are you planning to support x86 JVM's in the plugin or 64bit only?

**Comment by warmuuh** [comment]
if you want, you can compile it and do a pull request... i was actually happy to make this thing compile for win64 already... i dont know how to compile for 32bit(nor have the time to look into it...)

**Comment by menelaosbgr** [comment]
@warmuuh  @vkorobkov  
Hi, quick question: Where di I get the sass.dll file from? 
Are the binaries available? For an older project we are using JDK 1.6 so I am using 0.1.7-libsass_3.2.4 .
